tests/components/zwave_js/test_switch.py

```diff
@@ -0,0 +1,60 @@
+"""Test the Z-Wave JS sensor platform."""
+
+from .common import SWITCH_ENTITY
+
+
+async def test_switch(hass, binary_switch, integration, client):
+    """Test the switch."""
+    state = hass.states.get(SWITCH_ENTITY)
+
+    assert state
+    assert state.state == "off"
```

Comment:
We should exercise the state update also by emitting an event after the initial entity set up.